### PR TITLE
Fix for SOV-732

### PR DESCRIPTION
**Pull request by andkononykhin**

### CODE REVIEW

File: agent/test/start_scripts_test.py [hunk 1]
```diff
@@ -6,17 +6,26 @@
 
 
 def test_startApiServer():
-    run_script('startApiServer')
+    port_range_end = 8090 # XXX
```
Comment (rajeshkalaria80) [inline agent/test/start_scripts_test.py]: I am not getting the meaning of that comment # XXX, what is that?

File: agent/test/start_scripts_test.py [hunk 2]
```diff
@@ -6,17 +6,26 @@
...
+    for port in range(8080, port_range_end):
+        res = run_script('startApiServer', '0.0.0.0', str(port))
+        try:
+            assert res == 0, 'script failed'
+        except AssertionError:
+            if res != 2 or port == port_range_end: # 2 means address in use
+                raise
+        else:
```
Comment (rajeshkalaria80) [inline agent/test/start_scripts_test.py]: this else is not indented as it should be.

File: agent/test/start_scripts_test.py (hunk 1 above)
Comment (andkononykhin) [inline agent/test/start_scripts_test.py]: I wanted to mark that port range end could be discussed / changed later

File: agent/test/start_scripts_test.py [hunk 3]
```diff
@@ -6,17 +6,26 @@
 
 
 def test_startApiServer():
-    run_script('startApiServer')
+    port_range_end = 8090 # XXX
+    for port in range(8080, port_range_end):
```
Comment (rajeshkalaria80) [inline agent/test/start_scripts_test.py]: Somehow it feels lots or little confusing, this is what I can think of (there might be still better version of it which you may wanna try):

for port in range(8080, port_range_end):
	res = run_script('startApiServer', '0.0.0.0', str(port))
	if res == 2:  # 2 means address in use
		continue
	else:
		break
	
assert res == 0, 'script failed'
return

**Pull request merged**